sushi-config.yaml

```diff
@@ -7,7 +7,7 @@ canonical: http://ehealth.sundhed.dk/fhir
 name: eHealth Infrastructure
 title: eHealth Infrastructure
 status: active # draft | active | retired | unknown
-version: 3.3.0
+version: 3.2.0
 fhirVersion: 4.0.1
 copyrightYear: 2021+
 releaseLabel: ci-build # ci-build | draft | qa-preview | ballot | trial-use | release | update | normative+trial-use
```

Comment:
According to https://wiki/display/FUTJT/GDL0002+-+Release+procedure shouldn't the "releaseLabel" be updated? see 1.b